packages/pn-landing-webapp/src/layout/LandingLayout.tsx

```diff
@@ -22,6 +22,20 @@ const LandingLayout = ({ children }: Props) => {
   const { pathname } = useRouter();
   const appData = getAppData();
   const assistanceEmail = pathname !== "/pubbliche-amministrazioni" ? SEND_PF_HELP_EMAIL : PAGOPA_HELP_EMAIL;
+  const [windowURL, setWindowURL] = useState<string>();
+
+  // 
+  useEffect(() => {
```

Comment:
are the useEffect and useState necessary? I think that you can define a const windowURL = window ? window.location.origin : null

Reply:
I was inspired by the second response in https://stackoverflow.com/questions/55151041/window-is-not-defined-in-next-js-react-app, as suggested by @ZorbaDimatteo . I fear to access window in arbitrary places, since I don't fully understand which part of the code is executed server-side and which part browser-side.